### PR TITLE
working multi socket sesshhh

### PR DESCRIPTION
**Pull request by mattlemmone**
open 2 tabs on 4200, tada

### CODE REVIEW

File: web/src/index.js [hunk 1]
```diff
@@ -1,7 +1,29 @@
-const ws = new WebSocket(`ws://${location.hostname}:6999`)
+const ws = new WebSocket(`ws://${location.hostname}:6999?name=TESTBOI` + Math.random() * 100)
+
+function hi(msg){
```
Comment (mattlemmone) [inline web/src/index.js]: pls trash my heinous crime

**Comment by derekgaska** [comment]
idkgobutcoolcoolcoolcoolcoolsicksicksicksicksick

File: server/channel.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ func (c *Channel) EvictClients() {
 
 // Broadcast is used when sending a message to all clients in a Channel
 func (c *Channel) Broadcast(sender *Client, event *Event) {
-	log.Debugf("%s is sending a message: %+v", sender.name, event)
+	log.Debugf("%s is sending an event: %+v", sender.name, event)
```
Comment (justinjams) [inline server/channel.go]: 🤡 a message
👨‍🎨 an event

File: server/server.go [hunk 3]
```diff
@@ -18,28 +18,21 @@ type Server struct {
 
 // Address returns host:port fmt
 func (s *Server) Address() string {
-	return fmt.Sprintf("%s:%v", s.Host, s.Port)
+    return fmt.Sprintf("%s:%v",  s.Host, s.Port)
```
Comment (justinjams) [inline server/server.go]: TOO MANY WHITESPACE

File: server/server.go [hunk 4]
```diff
@@ -18,28 +18,21 @@ type Server struct {
...
-		log.Fatalf("failed to listen: %v", err)
-	}
-
-	s.listener = listener
-
 	s.httpServer = &http.Server{
+        Addr: s.Address(),
 		Handler:      http.HandlerFunc(handler),
 		ReadTimeout:  time.Second * 15,
 		WriteTimeout: time.Second * 15,
```
Comment (justinjams) [inline server/server.go]: heck to these giant tabs

File: web/README.md [hunk 5]
```diff
@@ -8,4 +8,4 @@ npm install
 npm run server
 
 # Play
-localhost:420
\ No newline at end of file
+localhost:4200
```
Comment (justinjams) [inline web/README.md]: wow you wanna blaze it big

File: web/server.js [hunk 6]
```diff
@@ -1,8 +1,13 @@
...
 
 app.use(express.static('public'))
 app.use(express.static('dist'))
 
-app.listen(port, () => console.log(`That's a spicy meatball on ${port}!`))
\ No newline at end of file
+app.listen(
+    port,
+    'localhost',
+    () => console.log(`That's a spicy meatball on ${host}:${port}!`)
```
Comment (justinjams) [inline web/server.js]: nice

File: web/src/index.js (hunk 1 above)
Comment (justinjams) [reply web/src/index.js]: 🗑

File: server/channel.go (hunk 2 above)
Comment (mattlemmone) [reply server/channel.go]: im dead

File: web/README.md (hunk 5 above)
Comment (mattlemmone) [reply web/README.md]: im tryna be a 10x'er

**Pull request merged**